_posts/2023-09-21-hybrid-search.md

```diff
@@ -0,0 +1,393 @@
+        <td>DBPedia</td>
+        <td>100.8</td>
+        <td>107.7</td>
+        <td>6.9</td>
+        <td>117</td>
+        <td>130.9</td>
+        <td>13.9</td>
+        <td>129.8</td>
+        <td>150.2</td>
+        <td>20.4</td>
+    </tr>
+    <tr>
+        <td>FiQA</td>
+        <td>53.9</td>
+        <td>56.9</td>
+        <td>3</td>
+        <td>61.9</td>
+        <td>65</td>
+        <td>3.1</td>
+        <td>64</td>
+        <td>67.7</td>
+        <td>3.7</td>
+    </tr>
+    <tr>
+        <td><b>% change vs. Boolean query </b></td>
+        <td colspan="3"><b>6.40%</b></td>
+        <td colspan="3"><b>6.96%</b></td>
+        <td colspan="3"><b>8.27%</b></td>
+    </tr>
+</table>
+
+As shown in the preceding table, hybrid search improves the result quality by 8&ndash;12% compared to keyword search and by 15% compared to natural language search. Simultaneously, a hybrid query exhibits a 6&ndash;8% increase in latency compared to a Boolean query when executing the same inner queries. 
+
+Our experimental findings indicate that the extent of improvement in score relevance is contingent upon the size of the sampled data. For instance, the most favorable results are observed when `size` is in the [100 .. 200] range, whereas larger values of `size` do not enhance relevance but adversely affect latency.
+
+## Conclusions
+
+In this blog, we have included series of experiments that show that hybrid search produces results that are very close to theoretical expectations and in most cases are better than the results of individual queries alone. Hybrid search comes with a certain increase in latency, which will be addressed in future versions (you can track the progress of this enhancement in [this GitHub issue](https://github.com/opensearch-project/neural-search/issues/279)).
+
+It’s important to remember that datasets have different parameters and for some datasets hybrid search may not consistently improve results. Conversely, experimenting with different parameters&mdash;for instance, selecting higher values of `k` or a different `space_type` for neural search&mdash;may lead to better results. 
+
+We’ve seen that some of the conclusions may be applied to most datasets:
+
+* For semantic search, hybrid query with normalization produces better results compared to neural search or text search alone.
+* The combination of `min-max` score normalization and `arithmetic_mean` score combination achieves the best results, compared to other techniques.
+* In most cases, increasing the value of `k` in k-NN data type leads to better results up to a certain point, but after that there is no increase in relevance. At the same time, high values of `k` increase search latency, so from our observations, it’s better to choose the value of `k` between 100 and 200. 
+* We have seen best results when `innerproduct` is specified as a space type for k-NN vector fields. This may be because our models were trained using the inner product similarity function.
```

Comment:
```suggestion
* The best results are produced when `innerproduct` is specified as a space type for k-NN vector fields. This may be because our models were trained using the inner product similarity function.
```